rust/tests/cli_tests.rs

```diff
@@ -55,15 +54,18 @@ fn ok_test(
     let stdout = &cmd.unwrap().stdout;
     let output = match str::from_utf8(stdout) {
         Ok(v) => v,
-        Err(e) => panic!("Invalid UTF-8 sequence: {}", e),
+        Err(e) => panic!("Invalid UTF-8 sequence: {e}"),
     };
-    println!("{}", output);
+    println!("{output}");
 
     assert!(output.contains(&driver_name));
     if !browser_version.is_empty() && output.contains("cache") {
         assert!(output.contains(&driver_version));
     }
-    assert!(!output.contains("Trying with latest driver version"));
+    assert!(
```

Comment:
Please revert this change (that message is not necessary/accurate).

Reply:
Hm but it is strange. In my machine that test failed until I installed firefox.